ElectronicObserver/Window/Tools/EquipmentUpgradePlanner/CostCalculation/EquipmentUpgradePlanCostEquipmentViewModel.cs

```diff
@@ -15,7 +15,7 @@ public EquipmentUpgradePlanCostEquipmentViewModel(EquipmentUpgradePlanCostItemMo
 		SubscribeToApis();
 		Update();
 	}
-
+	
```

Comment:
This can probably be deleted.

Reply:
should be fixed now

Reply:
I just meant the tab, not the whole empty line, that should stay.

Reply:
readded the new line